packages/elements/src/datetime-picker/index.ts

```diff
@@ -60,6 +60,9 @@ const INPUT_FORMAT = {
   DATETIME_SECONDS_AM_PM: 'dd-MMM-yyyy hh:mm:ss aaa'
 };
 
+const CALENDAR_ID = 'calendar';
```

Comment:
let's add a comment indicating these 2 IDs are public APIs. Updating them would be a breaking change.
```suggestion
// public API
const CALENDAR_ID = 'calendar';
```